src/components/pages/ChatPage/index.jsx

```diff
@@ -257,6 +262,19 @@ class ChatPage extends React.Component {
     }
   };
 
+  getUserByUid = (uid) => {
+    const users = this.state.users;
+    let result = null;
```

Comment:
In snek we separate const and lets.
```suggestion
    const users = this.state.users;
    
    let result = null;
```

Reply:
Makes sense, yeah.